# HG changeset patch
# User Gregory Szorc <gps@mozilla.com>
# Date 1532548171 25200
#      Wed Jul 25 12:49:31 2018 -0700
# Node ID be2dae870e083c6aa9723dff4758dc6d739dd818
# Parent  6cc127c38c45762ad47401558ad73fba8995f607
hgtemplates: index page changes

We've changed the index page in various ways.

diff --git a/hgtemplates/gitweb_mozilla/index.tmpl b/hgtemplates/gitweb_mozilla/index.tmpl
--- a/hgtemplates/gitweb_mozilla/index.tmpl	2024-12-05 00:59:16
+++ b/hgtemplates/gitweb_mozilla/index.tmpl	2024-12-05 01:18:28
@@ -21,13 +21,56 @@
     <tr>
         <td><a href="?sort={sort_name}">Name</a></td>
         <td><a href="?sort={sort_description}">Description</a></td>
-        <td><a href="?sort={sort_contact}">Contact</a></td>
         <td><a href="?sort={sort_lastchange}">Last modified</a></td>
         <td>&nbsp;</td>
         <td>&nbsp;</td>
     </tr>
     {entries%indexentry}
 </table>
+&nbsp;
+&nbsp;
+&nbsp;
+<div class="title">
+     Repository Layout
+</div>
+<table cellspacing="0">
+    <tr>
+        <td><a href="/">/</a></td>
+        <td>Mozilla top level repositories</td>
+    </tr>
+    <tr>
+        <td><a href="/build">build</a></td>
+        <td>Build team projects</td>
+    </tr>
+    <tr>
+        <td><a href="/conduit-testing">conduit-testing</a></td>
+        <td>Test repos that integrate with lower environments</td>
+    </tr>
+    <tr>
+        <td><a href="/experimental">experimental</a></td>
+        <td>Playground for version control wizards</td>
+    </tr>
+    <tr>
+        <td><a href="/hgcustom">hgcustom</a></td>
+        <td>Mercurial customizations</td>
+    </tr>
+    <tr>
+        <td><a href="/integration">integration</a></td>
+        <td>Source code integration work</td>
+    </tr>
+    <tr>
+        <td><a href="/projects">projects</a></td>
+        <td>Miscellaneous project repositories</td>
+    </tr>
+    <tr>
+        <td><a href="/releases">releases</a></td>
+        <td>Release branches (use releases/l10n-branchname for l10n repos)</td>
+    </tr>
+    <tr>
+        <td><a href="/SeaMonkey">SeaMonkey</a></td>
+        <td>The SeaMonkey Project</td>
+    </tr>
+</table>
 <div class="page_footer">
 {motd}
 </div>
